src/seedu/addressbook/Main.java

```diff
@@ -31,12 +32,13 @@
     private List<? extends ReadOnlyPerson> lastShownList = Collections.emptyList();
 
 
-    public static void main(String... launchArgs) {
+    public static void main(String... launchArgs) throws NoStorageFileException {
```

Comment:
You should handle the exception in your program, like raising a notification in UI to users, instead of throwing the exception out of the program.